Assets/VRTK/Scripts/Utilities/VRTK_PseudoSDKObject.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Pseudo SDK Object|Utilities|90063
+namespace VRTK
+{
+    using UnityEngine;
+
+    /// <summary>
+    /// The GameObject that the Pseudo SDK Object script is applied to will become a child of the selected SDK Object.
+    /// </summary>
+    public class VRTK_PseudoSDKObject : MonoBehaviour
+    {
+        /// <summary>
+        /// Valid SDK Objects
+        /// </summary>
+        /// <param name="Boundary">The main camera rig/play area object that defines the player boundary.</param>
+        /// <param name="Headset">The main headset camera defines the player head.</param>
+        /// <param name="LeftController">The main object that defines the left controller.</param>
+        /// <param name="RightController">The main object that defines the right controller.</param>
+        public enum SDKObjects
```

Comment:
This enum should probably be named `SDKObject`, since this is not usable as a bitfield (which would offer using _multiple_ ones).